cargo-dist/src/tasks.rs

```diff
@@ -2795,3 +2810,30 @@ impl InstallReceipt {
         })
     }
 }
+
+// Determines the *cached* HEAD for a submodule within the workspace.
+// Note that any unstaged commits, and any local changes to commit
+// history that aren't reflected by the submodule commit history,
+// won't be reflected here.
+fn submodule_head(submodule_path: &Utf8PathBuf) -> DistResult<String> {
+    let output = Cmd::new("git", "fetch expected latest commit for a submodule")
```

Comment:
This isn't the format of the error string, it's an explanation of what we were trying to do (gets prefixed with "failed to"). So it should be like "get submodule commit".

Reply:
Oh yeah, sorry, "expected" wasn't meant as an error message - it was because this fetches the cached/configured (eg "expected") commit of the submodule rather than what's currently there on disk

Reply:
Switched to the technical git term, "cached".